src/core/linter-rules/wpt-tests-exist.js

```diff
@@ -26,35 +27,22 @@ export async function run(conf) {
     return;
   }
 
-  const offendingElements = [];
-  const offendingTests = new Set();
-
   /** @type {NodeListOf<HTMLElement>} */
   const elems = document.querySelectorAll("[data-tests]");
   const testables = [...elems].filter(elem => elem.dataset.tests);
 
   for (const elem of testables) {
-    const tests = elem.dataset.tests
+    elem.dataset.tests
       .split(/,/gm)
       .map(test => test.trim().split("#")[0])
-      .filter(test => test);
-
-    const missingTests = tests.filter(test => !filesInWPT.has(test));
-    if (missingTests.length) {
-      offendingElements.push(elem);
-      missingTests.forEach(test => offendingTests.add(test));
-    }
+      .filter(test => test && !filesInWPT.has(test))
+      .map(missingTest => {
```

Comment:
Maybe group by element?

Reply:
I prefer them as they are for now. But will update them after I play around bit.